ui/plugins/job_details/main.html

```diff
@@ -5,6 +5,8 @@
       <span ng-if="line.content_type != 'TalosResult'" ng-switch on="line.content_type">
         <a ng-switch-when="link" title="{{line.value}}"
            href="{{line.url}}" target="_blank">{{line.value}}</a>
+
+        <a title="{{line.value}}" href="http://mozilla.github.io/wptview/#/?urls={{line.url | encodeURIComponent}},{{job_details[job_details.length - 1].value | encodeURIComponent}}" ng-if="line.value.endsWith('raw.log')"><br>Filter results in wptview</a>
```

Comment:
Putting it on a seperate line is indeed confusing. How about something like this:

```html
<span ng-if="line.value.endsWith('raw.log')">
- <a title="{{line.value}}" href="http://mozilla.github.io/wptview/#/?urls={{line.url | encodeURIComponent}},{{job_details[job_details.length - 1].value | encodeURIComponent}}" ng-if="line.value.endsWith('raw.log')">open in wptview</a>
</span>
```

I'm still not 100% sure the results are valid -- I can't seem to load anything from the URLs it generates:

http://mozilla.github.io/wptview/#/?urls=http:%2F%2Fmozilla-releng-blobs.s3.amazonaws.com%2Fblobs%2Fmozilla-inbound%2Fsha512%2Ff67c282527e4015e922f20eb71db5881e7d5448db052dfe05ce5e4f233bfd9803f7d5f1c832593793137a8f339cc2c950684c758186487bbf280f0b9e245e793,Rev7%20MacOSX%20Yosemite%2010.10.5%20mozilla-inbound%20debug%20test%20web-platform-tests-e10s-8

Am I missing something?